eos/b-decays/observables.cc

```diff
@@ -355,6 +355,46 @@ namespace eos
     }
     // }}}
 
+    // B_s -> K l nu
+    // {{{
+    ObservableGroup
+    make_bs_to_k_l_nu_group()
+    {
+        auto imp = new Implementation<ObservableGroup>(
+            R"(Observables in $B_s\to \bar{K} \ell^-\bar\nu$ decays)",
+            R"(The option "l" selects the charged lepton flavor.)"
+            R"(The option "form-factors" selects the form factor parametrization.)",
+            {
+                make_observable("B_s->Klnu::dBR/dq2", R"(d\mathcal{B}(B_s\to \bar{K}\ell^-\bar\nu)/dq^2)",
+                        Unit::InverseGeV2(),
+                        &BToPseudoscalarLeptonNeutrino::differential_branching_ratio,
+                        std::make_tuple("q2"),
+                        Options{ { "U", "u" }, {"q", "s"}, { "I", "1/2" } }),
```

Comment:
hmm why is the option "U" enforced here?

Reply:
Because it is a ``b->ulnu`` transition.

Reply:
Ah of course, sorry, I got confused